Source/WebCore/platform/cocoa/VideoFullscreenModelVideoElement.h

```diff
@@ -47,23 +47,19 @@ class HTMLVideoElement;
 class TextTrack;
 class PlaybackSessionModelMediaElement;
 
-class VideoFullscreenModelVideoElement : public VideoFullscreenModel, public EventListener {
+class VideoFullscreenModelVideoElement final : public VideoFullscreenModel {
 public:
     static RefPtr<VideoFullscreenModelVideoElement> create()
     {
         return adoptRef(*new VideoFullscreenModelVideoElement());
     }
-    WEBCORE_EXPORT virtual ~VideoFullscreenModelVideoElement();
+    WEBCORE_EXPORT ~VideoFullscreenModelVideoElement();
     WEBCORE_EXPORT void setVideoElement(HTMLVideoElement*);
     HTMLVideoElement* videoElement() const { return m_videoElement.get(); }
     WEBCORE_EXPORT RetainPtr<PlatformLayer> createVideoFullscreenLayer();
     WEBCORE_EXPORT void setVideoFullscreenLayer(PlatformLayer*, Function<void()>&& completionHandler = [] { });
-    WEBCORE_EXPORT void willExitFullscreen();
-    WEBCORE_EXPORT void waitForPreparedForInlineThen(Function<void()>&& completionHandler = [] { });
-    
-    WEBCORE_EXPORT void handleEvent(WebCore::ScriptExecutionContext&, WebCore::Event&) override;
-    void updateForEventName(const AtomString&);
-    bool operator==(const EventListener& rhs) const override { return static_cast<const WebCore::EventListener*>(this) == &rhs; }
+    WEBCORE_EXPORT void willExitFullscreen() override;
```

Comment:
Given the class is final, we probably want to use final in this method (and others as well).

Reply:
done